types/core/config.d.ts

```diff
@@ -26,29 +26,30 @@ declare const CONFIG: {
    */
   Item: {
     collection: Items;
-    entityClass: typeof Item;
-    sheetClasses: typeof ItemSheet;
+    entityClass: ConstructorOf<Item>;
+    sheetClass: ConstructorOf<ItemSheet>;
```

Comment:
Same as with the Actor config.

Reply:
The change here is correcting an apparent typo.